pkg/drivers/qemu/qemu.go

```diff
@@ -623,6 +643,12 @@ func (d *Driver) pidfilePath() string {
 	return filepath.Join(machineDir, "qemu.pid")
 }
 
+func (d *Driver) extraDiskPath(i int) string {
+	machineDir := filepath.Join(d.StorePath, "machines", d.GetMachineName())
+	diskFile := fmt.Sprintf("extra-disk-%d.raw", i)
+	return filepath.Join(machineDir, diskFile)
+}
```

Comment:
There is a helper for this pkg/drivers/common.go:ExtraDiskPath
using different name format ("%s-%d.rawdisk", d.GetMachineName(), diskID).
The helper is used by both kvm2 and hyperkit drivers.

Is there a reason to use diffrent name for the extra disk when using qemu2?
This can lead to confusing behavior - starting with kvm2 and then qemu2 will create 2 extra disks with different names, instead of reusing the existing disks.